catroid/src/main/res/values/strings.xml

```diff
@@ -1300,6 +1300,11 @@
     <string name="drone_set_tilt_outdoor">20 degrees </string>
     <string name="drone_set_tilt_max">30 degrees</string>
 
+    <!-- Notification Channel -->
+    <string name="channel_name">Pocket Code</string>
+    <string name="channel_description">Pocket Code notification channel</string>
```

Comment:
While you are at eliminating the hardcoded "Pocket Code" from strings: if it is easy, please also correct the other hardcoded strings in https://github.com/Catrobat/Catroid/blob/develop/catroid/src/main/res/values/strings.xml#L208-L209 => 

`<string name="error_scratch_converter_outdated_app_version">Your version of Pocket Code is outdated. Our Scratch
        converter only works on the latest version. Please update it through Google Play</string>
`

should be

`<string name="error_scratch_converter_outdated_app_version">Your version of this app is outdated. Our Scratch
        converter only works on the latest version. Please update the app.</string>
`

Please let me know whether it is possible. Thank you.

There are more eliminations of hardcoded names that are needed in strings.xml, e.g., concerning Google Play, but that will come with another pull request, independent of this one, so don't concern yourself with them in the present pull request.